### ISSUE TITLE
Investigate if splitting build steps into multiple build steps improves reusebility of the images

### ISSUE THREAD

**Issue by tkilias** [exasol/script-languages-release#289]
## Background 

- In https://github.com/exasol/script-languages-release/issues/245 was asked if we keep the docker build cache and if we can use it, that might be difficult
- However, exaslct already provides a mechanism for reusing images, but in certain cases it might be too coarse.
- Splitting up some build steps into multiple, might improve the reuse of images and with this, build speed.
- A example could be, if we install apt packages, python packages and r packages in the same build step, we could split this into three build steps. We could put them in on Pipeline or use a diamond pattern, where we first install the apt packages, have then two build steps in parallel which install the python and r-packages and at last copy the updates into one images.

**Comment by tkilias** [comment]
obsolete, happened already in past and we split the standard slc soon into multiple flavors.
